src/pages/CreateAccount/CreateAcc.module.scss

```diff
@@ -0,0 +1,92 @@
+.box {
+    padding: 120px 40px;
+    width: 100%;
+    background-color: cornflowerblue;
+  }
+  
+  .createAccountBtn {
+      text-decoration: none;
+    margin-top: 50px;
+    display: inline-block;
+    outline: 0;
+    cursor: pointer;
+    border: 2px solid #000;
+    border-radius: 3px;
+    color: #fff;
+    background: #000;
+    font-size: 20px;
+    font-weight: 600;
+    line-height: 28px;
+    padding: 12px 20px;
+    text-align: center;
+    transition-duration: 0.15s;
+    transition-property: all;
+    transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+    :hover {
+      color: #000;
+      background: rgb(255, 218, 87);
+    }
+  }
+  
+  .inputField {
+    width: 360px;
```

Comment:
repeated styles for input. Create geleral className for import with all repeatable styles and modify it by other classNames